### ISSUE TITLE
Conformance WCAG 3: Counting passing issues and documenting may be a poor use of a company's budget

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/silver#560]
As a manual tester, the idea of manually counting passing instances of issues is daunting. It would result in companies spending a lot of their A11Y budget on things they have done right.

I would like to suggest that manual counting and documenting of things companies do right be off the table.  There is no mainstream Q/A methodology that I am aware of, where Q/A testers document passing issues.

We could use AI to sample etc... but without an automated way to count all passing issues, I think we adjust our conformance model so that companies' budgets are not used that way.

**Comment by bruce-usab** [comment]
I do agree that the current model where auditors focus on problems seems pretty reasonable.

It is not much comfort to utility customers to be told how many days they had power last year!  OTOH, <q>off the table</q> might be too strong.  It does seem to me like 10000 images with good alt might result in some scoring advantage over a site without images.  I agree that this sort of nuance can be a lower priority than mot other things.  It also seems to me that a larger site -- which generally does a good job with alt text (and other accessibility issues at scale) -- would be in position to brag/report on that, without a wcag3 blessed metric.

**Comment by MakotoUeki** [comment]
I do agree with both of you. 

For alt text, I don't see any issues other than lack of concrete graphic examples for introducing various type of techniques. It is also true for any other existing SCs in WCAG 2.x as they are testable and repeatable. We can migrate them as it is into the WCAG 3's structure and templates. There might be some issues to be addressed though, such as adding examples, clarification or modification.

As the Alt Text subgroup lead, I'd like to share with everyone that measuring, testing and scoring sections for "Text Alternative" guideline in the FPWD were placeholders. We haven't discussed them as the subgroup. They were there to show how it will look like  in the new document template. It is not ready for the discussion like we did at the call.

For the discussion with "alt text" example, it might be more productive to start with the existing issue for testing SC 1.1.1 in WCAG 2.x. If there are no issues, I don't think we will need scoring for the text alternative in the first place. This can be true for all of the existing success criteria in WCAG 2 as they are testable requirements. 

And I don't see any reason for having the new scoring system for them. I have big concerns that it would result in bigger burden than WCAG 2.x. For any new guidelines which don't exist in WCAG 2 due to lack of testability and repeatability, the scoring might work. 

For instance, it might be possible for us to have a requirement for guidelines in WCAG 3 that each guideline must be testable and the test results must be repeatable as it has been in WCAG 2.x, OTHERWISE it must be scored if a guideline is not testable and repeatable.

And I agree with Bruce's point. We might need to take care of the nuance of accessibility level they've got if needed so that they could brag/report the nuance of how accessible it is.

Plus, please correct me if I'm misunderstanding or missing something as the premise for this discussion.

**Comment by jake-abma** [comment]
https://github.com/w3c/wcag/issues/2058#issuecomment-930197486

@mraccess77 wrote:

It's also much more complicated than simply telling a tool to count all img elements. We have img, role=img, svg, canvas, CSS background images, font icons, embed, object, video poster images, role doc-graphic, etc. Counting them all up is likely to not be accurate. If we do count them that information would need to be recorded as part of the audit and documented.

**Comment by jake-abma** [comment]
https://github.com/w3c/wcag/issues/2058#issuecomment-930218563

Also the loading of new images when scrolling the page will not help counting them... let alone the importance of images further away in the scroll.

**Comment by cwadamsoforacle** [comment]
Proposed response:  

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
